### PR TITLE
Update edit command

### PR DESCRIPTION
**Pull request by underthehai**
Update javadocs
Correct the command usage messages
Handle missing index cases

### CODE REVIEW

File: src/main/java/seedu/address/logic/wanderlustlogic/wanderlustcommands/edit/EditAccommodationCommand.java [hunk 1]
```diff
@@ -30,15 +31,16 @@
...
-            + "[" + PREFIX_LOCATION + "LOCATION] "
             + "Example: " + COMMAND_WORD + " 1 "
-            + PREFIX_NAME + "Hard Rock Hotel"
-            + PREFIX_START + " jan 20 2020 "
-            + PREFIX_END + " jan 30 2020"
-            + PREFIX_LOCATION + "Sentosa"
-            + PREFIX_COST + "SGD500";
+            + PREFIX_NAME + "Hard Rock Hotel "
+            + PREFIX_LOCATION + "Sentosa "
+            + PREFIX_COST + "$500 "
```
Comment (lyeyixian) [inline src/main/java/seedu/address/logic/wanderlustlogic/wanderlustcommands/edit/EditAccommodationCommand.java:41]: I think the cost is just numbers? iirc

File: src/main/java/seedu/address/logic/wanderlustlogic/wanderlustcommands/edit/EditAccommodationCommand.java [hunk 2]
```diff
@@ -30,15 +31,16 @@
             + ": Edits the accommodation identified by the index number used in the displayed accommodation list.\n"
             + "Parameters: INDEX (must be a positive integer)\n"
             + "[" + PREFIX_NAME + "NAME] "
+            + "[" + PREFIX_LOCATION + "LOCATION] "
```
Comment (lyeyixian) [inline src/main/java/seedu/address/logic/wanderlustlogic/wanderlustcommands/edit/EditAccommodationCommand.java:34]: Do you think we want to make the arrangements of all these fields consistent throughout our whole repo. Cos I feel its a little bit messy right now, everybody's arrangement is different. 
@jiaweiteo @jeannetoh99 @underthehai @timjkong

File: src/main/java/seedu/address/logic/wanderlustlogic/wanderlustcommands/edit/EditActivityCommand.java [hunk 3]
```diff
@@ -25,27 +24,25 @@
...
-            + PREFIX_NAME + "Hard Rock Hotel"
-            + PREFIX_IMPORTANCE + "2"
-            + PREFIX_LOCATION + "Sentosa"
-            + PREFIX_COST + "SGD500"
-            + PREFIX_START + "20 September 2020"
-            + PREFIX_END + "30 September 2020";
+            + PREFIX_NAME + "Ice Fishing "
+            + PREFIX_IMPORTANCE + "2 "
+            + PREFIX_LOCATION + "Ice Park "
+            + PREFIX_COST + "$50 "
```
Comment (lyeyixian) [inline src/main/java/seedu/address/logic/wanderlustlogic/wanderlustcommands/edit/EditActivityCommand.java:44]: Same as above

File: src/main/java/seedu/address/logic/wanderlustlogic/wanderlustcommands/edit/EditActivityCommand.java [hunk 4]
```diff
@@ -25,27 +24,25 @@
...
  * Edits an existing Activity in the address book and updates the travel plan/wishlist in the current directory
- * Edits the importance, cost, location, startdate, enddate
+ * An activity contains the field name, location, cost, level of importance and date time
  */
 public class EditActivityCommand extends EditCommand {
     public static final String COMMAND_WORD = "activity";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Edits the travel plan identified by the index number used in the displayed travel planner list.\n"
+            + ": Edits the activity identified by the index number used in the displayed travel planner list.\n"
```
Comment (lyeyixian) [inline src/main/java/seedu/address/logic/wanderlustlogic/wanderlustcommands/edit/EditActivityCommand.java:33]: Is it travel plan list?

File: src/main/java/seedu/address/logic/wanderlustlogic/wanderlustcommands/edit/EditCommand.java [hunk 5]
```diff
@@ -14,10 +14,17 @@
...
-            + "Example: " + COMMAND_WORD + " 1";
+    public static final String SPECIFY_INDEX = "Please Specify the index";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + " with optional fields\n"
+            + "Edits a travel plan to the travel planner using the format:\n"
+            + "edit -travelplan n/NAME sd/START_DATE ed/END_DATE\n"
+            + "Edits an activity to the current travel plan using the format:\n"
+            + "edit -activity n/NAME i/LEVEL_OF_IMPORTANCE l/LOCATION c/COST d/DATE_AND_TIME\n"
+            + "Edits an accommodation to the current travel plan using the format:\n"
+            + "edit -accommodation n/NAME l/LOCATION c/COST n/NIGHTS\n"
```
Comment (lyeyixian) [inline src/main/java/seedu/address/logic/wanderlustlogic/wanderlustcommands/edit/EditCommand.java:25]: I think it is suppose to be start date and end date?

**Review by lyeyixian — APPROVED** [review]
LGTM. Btw your PR title is wrong hahaha, can't have past tense. It is suppose to be Update

File: src/main/java/seedu/address/logic/wanderlustlogic/wanderlustcommands/edit/EditAccommodationCommand.java (hunk 1 above)
Comment (underthehai) [reply src/main/java/seedu/address/logic/wanderlustlogic/wanderlustcommands/edit/EditAccommodationCommand.java:41]: yes it is supposed to be numbers, i will change it

File: src/main/java/seedu/address/logic/wanderlustlogic/wanderlustcommands/edit/EditAccommodationCommand.java (hunk 2 above)
Comment (underthehai) [reply src/main/java/seedu/address/logic/wanderlustlogic/wanderlustcommands/edit/EditAccommodationCommand.java:34]: I tried to follow the order given in the UI

File: src/main/java/seedu/address/logic/wanderlustlogic/wanderlustcommands/edit/EditActivityCommand.java (hunk 4 above)
Comment (underthehai) [reply src/main/java/seedu/address/logic/wanderlustlogic/wanderlustcommands/edit/EditActivityCommand.java:33]: I am intending to specify it to "displayed activity list"!

File: src/main/java/seedu/address/logic/wanderlustlogic/wanderlustcommands/edit/EditCommand.java (hunk 5 above)
Comment (underthehai) [reply src/main/java/seedu/address/logic/wanderlustlogic/wanderlustcommands/edit/EditCommand.java:25]: yes, I will change it. thankss

**Pull request merged**